tests/test_cmd_strace.py

```diff
@@ -14,12 +14,21 @@
     "os.system('strace -V') != 0 or sys.platform in ['win32', 'cygwin']")
 class TestCmdStrace(object):
 
+    @staticmethod
+    def loader_for_task(task):
```

Comment:
I guess it would make more sense to just subclass ``TaskLoader2``instead of mocking everything...